app/components/Editor.js

```diff
@@ -125,13 +125,38 @@ export default class Editor extends React.Component {
       setEditorRawText: (rawText) => {
         // Called when the Bold Editor is loaded, when switching to a Bold
         // Editor note, or when uploading files, maybe in more places too.
+
+        const focused = this.redactor.editor.isFocus();
+        let point;
+
+        if (focused) {
+          // Maintain caret location if available.
```

Comment:
Btw we use [JSDoc](https://jsdoc.app/) style comments now, which look much cleaner. So this comment should look like:

```Javascript
/** Maintain caret location if available. */
```

And multiline comments would like:

```Javascript
/** Called when the Bold Editor is loaded, when switching to a Bold Editor note, 
or when uploading files, maybe in more places too. */
```

or 

```Javascript
/** 
 * Called when the Bold Editor is loaded, when switching to a Bold Editor note, 
 * or when uploading files, maybe in more places too. 
 */
```

Reply:
Sure, that looks good. I think I will do this conversion in a next PR because there are a lot of comments that can be updated to this format.